panther_analysis_tool/backend/public_api_client.py

```diff
@@ -329,23 +329,32 @@ def transpile_filters(
     def delete_saved_queries(
         self, params: DeleteSavedQueriesParams
     ) -> BackendResponse[DeleteSavedQueriesResponse]:
-        query = self._requests.delete_saved_queries()
-        delete_params = {
-            "input": {
-                "dryRun": params.dry_run,
-                "includeDetections": params.include_detections,
-                "names": params.names,
+        data: Dict = {"names": [], "detectionIDs": []}
+        # backend's delete function can only handle 100 IDs at a time,
```

Comment:
This restriction comes from Panther's API, and it's due to the DynamoDB limitation you mention bellow, right?